src/Avalonia.Controls/Platform/ITopLevelWithPlatformFeedback.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Avalonia.Controls
+{
+    public interface ITopLevelWithPlatformFeedback
```

Comment:
Note that even if ITopLevelImpl can support some feature, it should still be considered optional. A notable example is our global menu: TopLevel only exposes it if it's supported by the current DE.